filebeat/_meta/common.reference.p2.yml

```diff
@@ -240,6 +240,20 @@ filebeat.inputs:
   # Maximum size of the message received over UDP
   #max_message_size: 10240
 
+#------------------------------ TCP prospector --------------------------------
+# Experimental: Config options for the TCP input
+#- type: tcp
+  #enabled: false
+
+  # The host and port to receive the new event
+  #host: "localhost:9000"
```

Comment:
I remember @andrewkroh suggested to not have a default and require the use to specify it?

Reply:
We currently don't have any default on this value, I've provided an example hosts/port, but It's not a hard default.

```golang
var defaultConfig = config{
	ForwarderConfig: harvester.ForwarderConfig{
		Type: "tcp",
	},
	Config: tcp.Config{
		LineDelimiter:  "\n",
		Timeout:        time.Minute * 5,
		MaxMessageSize: 20 * 1024 * 1024,
	},
}
```

Following your comment, I did a small change and added a more explicit error for the host.